### DISCUSSION TITLE
There are outstanding issues with your GitHub account. Please contact GitHub Support for more information.

### DISCUSSION THREAD

**Discussion by sajibAdhi** [community/community#49494] [GitHub Education] [answered]
To reverify my academic affiliation to continue to qualify for GitHub Education benefits I was [submitting my information again.](https://education.github.com/discount_requests/application) But when I submit my information, it redirects me to the same [page](https://education.github.com/discount_requests/application). And there is this highlighted notification `There are outstanding issues with your GitHub account. Please contact GitHub Support for more information.`

**Comment by samus-aran** [comment]
Hi @sajibAdhi 

This message can be an indication of many things that only the Support team can answer and resolve. So please do contact [Account or billing issues](https://support.github.com/request) Support and they will explain the details of this message.

**Reply by sajibAdhi** [reply]
@samus-aran The support team guided me here!!

![Web capture_30-3-2023_204229_support github com](https://user-images.githubusercontent.com/60180521/228873245-a5b8a3f6-4150-43f0-be9e-71faae24616c.jpeg)

**Reply by samus-aran** [reply]
We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. Contacting [Account or billing issues](https://support.github.com/request) Support is still the most suitable place to raise your inquiry. The support team have been notified that your ticket has been closed in error so you'll hear from them soon. ⭐

**Reply by Wenrh2004** [reply]
@samus-aran I'm having the same problem, but the work order request remains unanswered!

**Reply by yzlltyyh** [reply]
@samus-aran I'm having the same problem, but the work order request remains unanswered!

**Reply by xdtxAM** [reply]
I'm having the same problem, but the work order request remains unanswered!

**Reply by samus-aran** [reply, marked as answer]
Hi all 👋 

We are seeing a higher volume of student developer pack inquiries coming through. We are working our way through addressing these as fast as possible. I can share that your request has been passed onto the GitHub Education team and will be processed as soon as possible, but I am afraid I do not have a timeline to share for when you can expect this to happen.

Usually with these types of tickets we may include an investigation as fraudulent activity has been identified on either the account or the associated school's. We thank you for your patience and understanding as we do our due diligence ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

**Comment by devilshezi0** [comment]
components/search/SearchResults.tsx

**Reply by devilshezi0** [reply]
components/search/SearchResults.tsx

**Reply by devilshezi0** [reply]
> components/search/SearchResults.tsx

**Comment by devilshezi0** [comment]
components/search/SearchResults.tsx
